changelog/953.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Fixed a bug in the Particle Tracker simulation. The integrator had a sign
+error, thus the ExB drift came out in the incorrect direction.
```

Comment:
```suggestion
error, thus the E×B drift came out in the incorrect direction.
```
Unicode!

Reply:
That's some good application of unicode, I love it! But I think it's confusing enough for the mobile GitHub app that I can't merge the suggestion without a crash... Will merge after reporting it:D